sdk/storage/azure-storage-queue/azure/storage/queue/_models.py

```diff
@@ -261,13 +336,26 @@ def _from_generated(cls, generated):
 class MessagesPaged(PageIterator):
     """An iterable of Queue Messages.
 
-    :param callable command: Function to retrieve the next page of items.
+    :param Callable[[str], bytes] command: Function to retrieve the next page of items.
```

Comment:
Wanted to check on this type (my notes were not clear whether bytes return type was only applicable to the `Callable` changes in `_encryption.py` or also applicable here)